pkg/api/v1alpha1/tinkerbelltemplateconfig_defaults.go

```diff
@@ -156,8 +156,8 @@ func withDisableCloudInitNetworkCapabilities(b v1alpha1.VersionsBundle, disk str
 	}
 }
 
-func withTinkCloudInitAction(b v1alpha1.VersionsBundle, disk string, tinkerbellIp string) ActionOpt {
-	metadataString := fmt.Sprintf("\"http://%s:50061\"", tinkerbellIp)
+func withTinkCloudInitAction(b v1alpha1.VersionsBundle, disk string, tinkerbellLocalIp, tinkerbellLBIp string) ActionOpt {
+	metadataString := fmt.Sprintf("\"http://%s:50061\", \"http://%s:50061\"", tinkerbellLocalIp, tinkerbellLBIp)
```

Comment:
A brief comment here would be nice explaining how multiple urls work in the metadata_urls and how local IP vs. LB IP are relevant in bootstrap cluster vs. target cluster cases respectively

Reply:
Will add that 👍